test-requirements.txt

```diff
@@ -1,3 +1,4 @@
 flake8
```

Comment:
What version of flake8 are you using locally? Maybe try updating your version of flake8 and pin the version here, so you are sure you have the same version as on Travis.

Also make sure you are running everything with Python 3

Reply:
I'm on flake8 3.5.0 and everything runs on Python3